src/bundle/Resources/config/services/strategies.yaml

```diff
@@ -9,3 +9,11 @@ services:
     Ibexa\AdminUi\Strategy\ContentTypeThumbnailStrategy:
         tags:
             - { name: ibexa.repository.thumbnail.strategy.content, priority: -100 }
+
+    Ibexa\AdminUi\Strategy\FocusMode\ContentStructureRedirectStrategy:
+        tags:
+            - { name: ibexa.admin_ui.focus_mode.redirect_strategy, priority: -100 }
+
+    Ibexa\AdminUi\Strategy\FocusMode\OriginalPathRedirectStrategy:
+        tags:
+            - { name: ibexa.admin_ui.focus_mode.redirect_strategy, priority: -100 }
```

Comment:
`Content`-based strategy should always go first, so adjust priority accordingly. `-90`?